src/cli/onefuzz/status/cache.py

```diff
@@ -64,29 +65,34 @@ def fmt(data: Any) -> Any:
     raise NotImplementedError(type(data))
 
 
+class JobFilter(BaseModel):
+    job_id: Optional[List[UUID]]
+    project: Optional[List[str]]
+    name: Optional[List[str]]
+
+
 class TopCache:
-    JOB_FIELDS = (["Updated", "State", "Job", "Name", "Files"], [1, 1, 1, 1, "100%"])
-    TASK_FIELDS = (
-        [
-            "Updated",
-            "State",
-            "Job",
-            "Task",
-            "Type",
-            "Name",
-            "Files",
-            "Pool",
-            "Time left",
-        ],
-        [1, 1, 1, 1, 1, 1, 1, 1, "100%"],
-    )
-    POOL_FIELDS = (
-        ["Updated", "Pool", "Name", "OS", "State", "Nodes"],
-        [1, 1, 1, 1, 1, "100%"],
-    )
-
-    def __init__(self, onefuzz: "Onefuzz"):
+    JOB_FIELDS = ["Updated", "State", "Job", "Name", "Files"]
+    TASK_FIELDS = [
+        "Updated",
+        "State",
+        "Job",
+        "Task",
+        "Type",
+        "Name",
+        "Files",
+        "Pool",
+        "Time left",
+    ]
+    POOL_FIELDS = ["Updated", "Pool", "Name", "OS", "State", "Nodes"]
+
+    def __init__(
+        self,
+        onefuzz: "Onefuzz",
```

Comment:
Not new to this PR, but how about we edit this to `"OneFuzz"`?

Reply:
It needs to be the type name, which is Onefuzz.   I don't want to change the name of the type to OneFuzz, as that's a pretty large breaking change for the SDK.

Reply:
Ahh, of course. I was totally mis-reading this as a default value.

Reply:
One of the difficulties in the 'late type binding as strings'